### PR TITLE
Update monRadSummary, add monOznMonSummary

### PR DESCRIPTION
**Pull request by EdwardSafford-NOAA**
Recent changes/additions to gfs plot templates.

### CODE REVIEW

**Review by CoryMartin-NOAA — DISMISSED** [review]

**Comment by EdwardSafford-NOAA** [comment]
I've added the ozn yaml templates and updated the rad templates for recent eva graphics changes.

File: parm/gfs/monOznTime.yaml
Comment (CoryMartin-NOAA) [inline parm/gfs/monOznTime.yaml:23]: Just making a note here that one of the first things we should develop is the YAML generator using templating leveraging what we have in the global-workflow already

**Review by CoryMartin-NOAA — APPROVED** [review]

**Comment by CoryMartin-NOAA** [comment]
@EdwardSafford-NOAA should we wait for another review or just merge?

File: parm/gfs/monOznTime.yaml
Comment (EdwardSafford-NOAA) [reply parm/gfs/monOznTime.yaml:23]: Yes, that was one of the things I think we should discuss on Thurs.

**Comment by EdwardSafford-NOAA** [comment]
I think it's ok to merge these.  I know they will need more work, which will become clearer when we implement the yaml generator.

**Pull request merged**
